src/styles/Components/src/tokens-alert.css

```diff
@@ -0,0 +1,82 @@
+:root {
+  /**
+  * @tokens Alerts
+  * @presenter BorderRadius
+  */
+  --denhaag-alert-border-radius: var(--denhaag-border-radius);
+
+  /**
+  * @tokens Alerts
+  * @presenter Spacing
+  */
+  --denhaag-alert-padding-block-start: 24px;
+  --denhaag-alert-padding-block-end: 24px;
+  --denhaag-alert-padding-inline-start: 24px;
+  --denhaag-alert-padding-inline-end: 24px;
+
+  /**
+  * @tokens Alerts
+  * @presenter Spacing
+  */
+  --denhaag-alert-action-focus-border-color: var(--denhaag-ocher-4);
+  --denhaag-alert-action-focus-border-width: var(--denhaag-focus-border-width);
+  --denhaag-alert-action-focus-border-style: var(--denhaag-focus-border-style);
+  --denhaag-alert-action-focus-border-offset: 3px;
+
+  /**
+  * @tokens Alerts
+  * @presenter Spacing
+  */
+  --denhaag-alert-content-margin-inline-start: 24px;
+  --denhaag-alert-content-margin-inline-end: 24px;
+
+  /**
+  * @tokens Alerts
+  * @presenter Spacing
+  */
+  --denhaag-alert-action-padding-block-start: 16px;
+
+  /**
+  * @tokens Alerts
+  * @presenter Color
+  */
+  --denhaag-alert-error-background-color: var(--denhaag-red-1);
+  --denhaag-alert-error-title-color: var(--denhaag-red-5);
+  --denhaag-alert-error-text-color: var(--denhaag-red-5);
+  --denhaag-alert-error-icon-color: var(--denhaag-red-3);
+  --denhaag-alert-error-action-button-color: var(--denhaag-red-3);
```

Comment:
I don't have a very strong opinion, but since this design token is used to change the css property `background-color` for the action button, I'd prefer seeing the token called `...action-button-background-color`. EDIT: Which reminds me, there's no token for the text itself in the action button is there?

Reply:
Good point, will change it to background-color.
Indeed there's no token for the text in the action button, in figma its white on every alert variant.
For completeness, I will add a design token for every variant and set them to white in the component tokens.